mycli/main.py

```diff
@@ -793,9 +793,10 @@ def format_output(self, title, cur, headers, expanded=False,
             rows = list(cur)
             formatted = self.formatter.format_output(
                 rows, headers, format_name='vertical' if expanded else None)
+            first_line = formatted[:formatted.find('\n')]
```

Comment:
I guess splitting on `\n` and finding a longest line would be a bad idea because of performance?